kv/memberlist/memberlist_client.go

```diff
@@ -428,7 +442,12 @@ func (m *KV) starting(_ context.Context) error {
 	if err != nil {
 		return fmt.Errorf("failed to create memberlist: %v", err)
 	}
-
+	// Spawn notify workers.
+	numNotifyWorkers := runtime.NumCPU()
```

Comment:
what if multiple instances of this process run on the same physical node? afaik we tend to run on k8s without CPU limits, so `runtime.NumCPU()` will return the number of physical cores, right?
wouldn't this mean that if 2 pods run on the same physical node then each pod will create one worker per physical cpu? Maybe it would make sense to make that configurable instead?

Reply:
resolving, as with new per-key goroutine implementation this does not longer apply.